api/datastore/sql/sql_test.go

```diff
@@ -22,12 +24,7 @@ func newWithMigrations(ctx context.Context, url *url.URL) (*sqlStore, error) {
 		return nil, err
 	}
 
-	m, err := migrator(url.String())
-	if err != nil {
-		return nil, err
-	}
-
-	err = m.Down()
+	err = migrations.DownByOne(url.Scheme, ds.db.DB)
```

Comment:
`m.Down()` went all the way down, which then let the subsequent opening of the db go all the way back up. this ensures all migrations work (against all sql stores, except sqlite3 which doesn't like `DROP COLUMN`).

Reply:
Okay, can adjust that.